templates/index.html

```diff
@@ -1,25 +1,33 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <title>To-Do List</title>
 </head>
+
 <body>
     <h1>My To-Do List</h1>
     <a href="/uncompleted">Show Uncompleted Tasks Only</a>
     <a href="/">Show All Tasks</a>
+
+    <div>
+        <p>Total Tasks: {{ tasks|length }} | Completed: {{ tasks|selectattr('completed')|list|length }}</p>
```

Comment:
Total number of tasks

Reply:
The counts will automatically update whenever tasks are added, completed, or deleted, and will reflect the current filtered view (when showing only uncompleted tasks).